backend/databases/sql.py

```
@@ -0,0 +1,216 @@
+    conn.commit()
+    conn.close()
+    
+    return item_id
+
+def search_items(query: str, user_id: int):
+    """Search items - CONTAINS DELIBERATE VULNERABILITIES"""
+    conn = sqlite3.connect(DATABASE_PATH)
+    cursor = conn.cursor()
+    
+    # DELIBERATE SQL INJECTION VULNERABILITY
+    search_query = f"SELECT * FROM items WHERE name LIKE '%{query}%' AND user_id = {user_id}"
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)